app-emulation/vagrant/files/vagrant-1.9.7

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#
+# This is a wrapper to properly execute Vagrant within the embedded
+# Vagrant installation directory. This sets up proper environmental variables
+# so that everything loads and compiles to proper directories.
+
+VAGRANT_DIR="$( ruby -e 'print Gem::default_path[-1] + "/gems/vagrant-1.9.7"' )"
```

Comment:
By the way, are you really going to introduce a new script with different version number for every version? This is a real waste of space. How about using `vagrant.in` and using sed to substitute the version?